user/models.py

```diff
@@ -52,12 +52,15 @@ def to_json(self):
 
     @classmethod
     @transaction.atomic
-    def create(cls, access_token=None, **kwargs):
-        username = kwargs.get("sub")
+    def create(cls, **kwargs):
+        username = kwargs.get("email")
```

Comment:
we still want the `sub` here.
The email is not unique (2 Identity providers may provide the same email), the sub is unique